.github/workflows/checks.yaml

```diff
@@ -164,6 +164,18 @@ jobs:
       # use commit instead of ref so we can "go get" specific sdk version
       platform-ref: ${{ github.event.pull_request.head.sha || github.sha }}
 
+  # test latest otdfctl CLI 'main' against platform PR branch
+  otdfctl-test:
+    name: otdfctl e2e tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: opentdf/platform/test/start-up-with-containers@main
+        with:
+          platform-ref: ${{ github.event.pull_request.head.sha || github.sha }}
+      - uses: opentdf/otdfctl/e2e@feat/reusable-plat
```

Comment:
ref main after [PR is merged in otdfctl](https://github.com/opentdf/otdfctl/pull/369) and before merging